docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_extended_component_list.htm

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Extended Wearable UI Components</title>
+</head>
+<body onload="prettyPrint()" marginwidth="0" marginheight="0" id="content">
+
+<h1>Extended Wearable UI Components</h1>
+
+<p>The Web UI Framework (called as <b>TAU</b>; Tizen Advanced UI framework) provides rich Tizen components that are optimized for the Tizen Web browser.</p>
```

Comment:
Suggestion :
Ho about adding one simple line telling the difference between UI component and Extended UI component?
For example "You can also use some components with external libraries." or like that.

Reply:
ditto.